tests/unit/records/pandas/test_prep_for_csv.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+import unittest
+from records_mover.records.pandas import format_df_for_csv_output
+from records_mover.records.schema import RecordsSchema
+from records_mover.records import DelimitedRecordsFormat
+
+
+class TestFormatForCsv(unittest.TestCase):
+    def test_format_df_for_csv_output(self):
+        schema_data = {
+            'schema': "bltypes/v1",
+            'fields': {
+                "date": {
+                    "type": "date",
+                    "index": 1,
+                },
+                "time": {
+                    "type": "time",
+                    "index": 2,
+                },
+                # TODO:
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker